setup.cfg

```diff
@@ -68,4 +68,54 @@ fail-under = 8
 
 [pylint.typecheck]
 generated-members=torch.*, cv2.*
-good-names=x, y, z, i
\ No newline at end of file
+good-names=x, y, z, i
+
+[flake8]
```

Comment:
So I looked it up and seems that `flake8` doesn't have `pyproject.toml` support. However, `pylint` offers full support for it. Any reason to prefer flake over pylint? I've personally only used pylint so I am not aware of flake functionality. From what I can see in the config, we should be able to move everything over to pylint

Reply:
I have just used `flake8` until now, but I `pylint` is fine with me. Support of `pyproject.toml` is a big plus.

Reply:
Yes, in that case lets move to `pylint`. The `pyproject.toml` support will unify everything